app/api/csv/importEntity.ts

```diff
@@ -1,18 +1,23 @@
+/* eslint-disable max-statements */
```

Comment:
this should only be allowed on old, files, never added unless there is a really good reason.

Reply:
removed.